### PR TITLE
Feature/en 3 refactoring code

### PR DESCRIPTION
**Pull request by A-Romany**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    </build>
+    <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.telegram/telegrambots -->
+        <dependency>
+            <groupId>org.telegram</groupId>
+            <artifactId>telegrambots</artifactId>
+            <version>4.7</version>
+
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.h2database/h2 -->
```
Comment (A-Romany) [inline pom.xml]: I'm not sure if it is really needed here

Comment (AgienkoSergiy) [reply pom.xml]: It is not needed but it will be removed in future

File: src/main/java/Main.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import org.telegram.telegrambots.ApiContextInitializer;
+//import org.telegram.telegrambots.TelegramBotsApi;
```
Comment (AgienkoSergiy) [inline src/main/java/Main.java]: Cleanup these lines please

File: src/main/java/persistence/WordPool.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package persistence;
+
+public class WordPool {
```
Comment (AgienkoSergiy) [inline src/main/java/persistence/WordPool.java]: Remove it class please, cause it is not what this task about

File: src/main/java/service/EnglishBotService.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package service;
+
+import persistence.EnglishBotDAO;
+
+public class EnglishBotService {
```
Comment (AgienkoSergiy) [inline src/main/java/service/EnglishBotService.java]: Remove also this please

File: src/main/resources/hibernate.cfg.xml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (AgienkoSergiy) [inline src/main/resources/hibernate.cfg.xml]: And this file should be removed

File: src/main/resources/hibernate.properties [hunk 6]
```diff
@@ -0,0 +1,19 @@
+## Driver class for DB connection to DB
+hibernate.connection.driver=org.h2.Driver
```
Comment (AgienkoSergiy) [inline src/main/resources/hibernate.properties]: Remove this one too

**Review by AgienkoSergiy — CHANGES_REQUESTED** [review]

File: src/main/java/persistence/EnglishBotDAO.java [hunk 7]
```diff
@@ -0,0 +1,20 @@
+package persistence;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class EnglishBotDAO {
```
Comment (AgienkoSergiy) [inline src/main/java/persistence/EnglishBotDAO.java]: This class is not needed for now, since we have no DB

**Review by AgienkoSergiy — CHANGES_REQUESTED** [review]

**Review by AgienkoSergiy — APPROVED** [review]

**Pull request merged**